extensions/wikia/ContentReview/ContentReview.i18n.php

```diff
@@ -59,6 +59,10 @@
 	'content-review-status-in-review' => 'In review',
 	'content-review-status-approved' => 'Approved',
 	'content-review-status-rejected' => 'Rejected,',
+
+	'content-review-feedback-link-text' => 'Provide feedback',
+	'content-review-rejection-explanation' => '==Submitted script change was rejected==
```

Comment:
Unfortunately functionality for prefilling edit doesn't allow to pass parameters...
https://github.com/Wikia/app/blob/CE-2550-talk-page-link/extensions/wikia/CreatePage/CreatePage.php#L126-126
Spoke to George, and he accepted general message.